tests/subsys/caf/sensor_manager/prj.conf

```diff
@@ -9,6 +9,7 @@ CONFIG_ZTEST=y
 
 CONFIG_CAF=y
 CONFIG_CAF_SENSOR_MANAGER=y
+CONFIG_CAF_SENSOR_MANAGER_THREAD_PRIORITY=-1
```

Comment:
@MarekPieta  wasn't that the main assumption that sensor manager must work as preemptive thread?

Reply:
It should in general run in a preemptive thread, but if we would like to ensure that `sensor_events` are submitted precisely in given timeslots, then I have nothing against using higher thread priority here (that's why thread priority is configurable). Alternatively we would need to ensure that this thread would not be preempted even if it has low priority.